content/js/users.md

```diff
@@ -152,6 +152,27 @@ skygear.discover('chpapa').then((ben) => {
 });
 ```
 
+<a name="user-profile"></a>
+## User Profile
+
+Whenever a new user signs up, a user profile is automatically created for
+you to track user information other than their username, email or password.
+You can access the user profile the same way as accessing a record.
+
```

Comment:
perhaps adding that the "user profile" is a database table called `user`, but not the reserved `_user` table?